packages/core/upload/admin/src/components/TableList/PreviewCell.js

```diff
@@ -1,69 +1,81 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { prefixFileUrlWithBackendUrl, pxToRem } from '@strapi/helper-plugin';
-import { Avatar } from '@strapi/design-system/Avatar';
-import { Flex } from '@strapi/design-system/Flex';
-import { Typography } from '@strapi/design-system/Typography';
+import { getFileExtension, prefixFileUrlWithBackendUrl, pxToRem } from '@strapi/helper-plugin';
+import { Avatar, Initials } from '@strapi/design-system/Avatar';
+import { Box } from '@strapi/design-system/Box';
 import { Icon } from '@strapi/design-system/Icon';
 import Folder from '@strapi/icons/Folder';
 
-const GenericAssetWrapper = styled(Flex)`
-  span {
-    /* The smallest fontSize in the DS is not small enough in this case */
-    font-size: ${pxToRem(10)};
+import { AssetType } from '../../constants';
+import { createAssetUrl } from '../../utils';
+import { VideoPreview } from '../AssetCard/VideoPreview';
+
+const VideoPreviewWrapper = styled(Box)`
+  figure {
+    width: 26px;
+    height: 26px;
```

Comment:
These should be either `24px` (theme.spaces[6]) or `32px` (theme.spaces[7]) because the grid is based on 4.

Reply:
it's the size of the `Avatar` 
<img width="400" alt="image" src="https://user-images.githubusercontent.com/71838159/212910820-e434bd6d-54a4-43f1-8849-186922c321c7.png">
that would be strange to have a different "Preview" size for video than any other elements (image, file) - wdyt?

Reply:
Could we move both then to use either one of these values? :) @maevalienard wdyt?

Reply:
as this would require a second DS release (this PR was already depending on a previous DS release for `Avatar` and then `Initials`) and it seems not directly linked to ML listview conversation but more Avatar DS conversation - wdyt if I bring this to a ticket discussion and fix this in a follow up PR?

Reply:
32px would be fine to me :) (I made tests on both the mainNav and the table that are using this component)

Reply:
Yeah sure. Could you create a follow-up ticket in the DS repo, @ronronscelestes ?

Reply:
https://github.com/strapi/design-system/issues/844